### ISSUE TITLE
Can't use Dangerzone: Unknown error code '0'

### ISSUE THREAD

**Issue by rtfmkiesel** [freedomofpress/dangerzone#1025]
### What happened?

Every file I try to pass through dangerzone results in "error code '0'". I've tried the filetypes pdf, doc, docx and png.

### Linux distribution

Ubuntu 22.04 LTS via Pop!_OS

### Dangerzone version

0.8.0

### Document conversion logs

```shell
dangerzone-cli --output-filename test.pdf test.doc
 
╭──────────────────────────╮
│           ▄██▄           │
│          ██████          │
│         ███▀▀▀██         │
│        ███   ████        │
│       ███   ██████       │
│      ███   ▀▀▀▀████      │
│     ███████  ▄██████     │
│    ███████ ▄█████████    │
│   ████████████████████   │
│    ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀    │
│                          │
│    Dangerzone v0.8.0     │
│ https://dangerzone.rocks │
╰──────────────────────────╯
Assigning ID 'gbl7Su' to doc '/home/me/Downloads/test.doc'

Converting document to safe PDF
> /usr/bin/podman run --log-driver none --security-opt no-new-privileges --security-opt seccomp=/usr/share/dangerzone/seccomp.gvisor.json --cap-drop all --cap-add SYS_CHROOT --security-opt label=type:container_engine_t --network=none -u dangerzone --rm -i --name dangerzone-doc-to-pixels-gbl7Su dangerzone.rocks/dangerzone /usr/bin/python3 -m dangerzone.conversion.doc_to_pixels
[doc gbl7Su] 0% Converting page 1/1 from pixels to PDF
ERROR [doc gbl7Su] 0% Unknown error code '0'

Failed to convert document(s)
/home/me/Downloads/test.doc
```
```


### Additional info

+ Happens on the version installed via `apt` and on a self-built version using the dev env guide.
+ Happens on the GUI and the CLI.

**Comment by almet** [comment]
Hi, and thanks for opening a bug report here.

Can you please include the result of the following podman commands?

```bash
podman version
podman info -f 'json'
podman images
podman run hello-world
```

We support Ubuntu, but do not explicitly support Pop!OS (which is based on Ubuntu), so it might be that you're encountering a situation where the podman configuration there is the culprit.

**Comment by rtfmkiesel** [comment]
Would have to censor a lot from `podman info`, as quite a few things in this output are sensitive. That's why I did not include it from the beginning. 

`podman version` give the same output as inside an Ubuntu VM.  
`podman images` just list all the images including Dangerzone.  
`podman run hello-world` works as expected. 

Closing this issue, apparently a Pop!OS issue, works on Fedora as expected.
